src/Common/src/Interop/Windows/Kernel32/Interop.DCB.cs

```diff
@@ -9,13 +9,59 @@ internal partial class Interop
 {
     internal partial class Kernel32
     {
+        internal static class DCBFlags
```

Comment:
I prefer having public access modifiers on internal parents but I understand that you followed the existing style.